ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/management_api_gnmi.schema.yml

```
@@ -39,11 +39,70 @@ keys:
                 ip_access_group:
                   type: str
                   description: ACL name
+          grpc_tunnel:
```

Let's use plural as we create a list of them
```suggestion
          grpc_tunnels:
```